pkg/build/builder/sti.go

```diff
@@ -259,7 +264,11 @@ func (s *S2IBuilder) Build() error {
 		return errors.New(buffer.String())
 	}
 
-	glog.V(4).Infof("Creating a new S2I builder with build config: %#v\n", describe.Config(config))
+	client, err := docker.NewEngineAPIClient(config.DockerConfig)
```

Comment:
sorry the git diff tricked me into thinking both of these changes were in the same function and thus someone would expect to reuse the same client object.